config/routes.rb

```diff
@@ -19,7 +19,12 @@
 
   get 'workflows/:workflow_id/extractors/:extractor_id/extracts', to: 'extracts#index'
   put 'workflows/:workflow_id/extractors/:extractor_id/extracts', to: 'extracts#update'
+  get 'workflows/:workflow_id/extractors/export', to: 'data_requests#request_extracts'
 
   get 'workflows/:workflow_id/reducers/:reducer_id/reductions', to: 'reductions#index'
   put 'workflows/:workflow_id/reducers/:reducer_id/reductions', to: 'reductions#update'
+  get 'workflows/:workflow_id/reducers/export', to: 'data_requests#request_reductions'
+
+  get 'data_request/:request_id', to: 'data_requests#check_status'
```

Comment:
Did you consider doing just a `resources :data_requests`, and letting the API user specify the type via the JSON body that's posted in, rather than various non-REST routes?